jbpm-bootstrap-kjar/readme.md

```diff
@@ -0,0 +1,4 @@
+Repository Init Content
```

Comment:
Maybe we can add a brief description of this module in this file. Also, please capitalize the name of the file.